modules/dsp/chowdsp_dsp_data_structures/Other/chowdsp_SmoothedBufferValue.h

```diff
@@ -55,7 +55,7 @@ class SmoothedBufferValue
      */
     void prepare (double sampleRate, int samplesPerBlock, bool useInternalVector = true);
 
-    /** Resets the state of the smoother with a given value. */
+    /** Resets the state of the smoother with a given value. Expects the value in linear gain */
```

Comment:
Whether or not this method expects a linear gain value depends on what kind of value is being stored in the smooth buffer... it could be a Decibel value, or it might not be a gain value at all!